configs/kubernetes-rules-configmap.yaml

```diff
@@ -846,3 +846,79 @@ data:
           branch: release-1.12
         - repository: client-go
           branch: release-9.0
+    - destination: sample-cli-plugin
+      library: false
+      branches:
+      - source:
+          branch: master
+          dir: staging/src/k8s.io/sample-cli-plugin
+        name: master
+        dependencies:
+        - repository: api
+          branch: master
+        - repository: apimachinery
+          branch: master
+        - repository: cli-runtime
+          branch: master
+        - repository: client-go
+          branch: master
+    - destination: cli-runtime
```

Comment:
order is wrong: it must be in topological order, i.e. depending repos after dependencies.

Reply:
Fixed